### PR TITLE
add missing dependency

### PR DESCRIPTION
**Pull request by stuartnelson3**
adds missing dependency that causes breakout graphs to break

### CODE REVIEW

**Comment by stuartnelson3** [comment]
@juliusv threat level 1..

**Comment by juliusv** [comment]
:+1:

**Comment by stuartnelson3** [comment]
currently failing because semaphore has an incorrect phantomjs version

File: spec/javascripts/angular/controllers/single_widget_ctrl_spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  beforeEach(inject(function(_$controller_, $rootScope){
+    $controller = _$controller_;
+    scope =  $rootScope;
+  }));
+
+  it('loads correctly', function() {
+    var $scope = scope.$new(true);
+    var controller = $controller('SingleWidgetCtrl', { $scope: $scope });
+    expect($scope.servers).toEqual([]);
+    expect($scope.globalConfig.aspectRatio).not.toBe(undefined);
```
Comment (grobie) [inline spec/javascripts/angular/controllers/single_widget_ctrl_spec.js]: Did these tests catch the issue? Bonus points for red-green testing (http://en.wikipedia.org/wiki/Test-driven_development#Development_style).

Comment (stuartnelson3) [inline spec/javascripts/angular/controllers/single_widget_ctrl_spec.js]: yeah they did, gave the same missing dependency error that was reported. this is really just a smoke test to make sure everything runs.

Comment (grobie) [inline spec/javascripts/angular/controllers/single_widget_ctrl_spec.js]: This is great! Ideally all bug reports will be first written as tests which fail and then fixed (red-green testing). :lollipop: work :)

**Pull request merged**